src/import.ml

```diff
@@ -43,6 +43,12 @@ let show_message kind fmt =
       ())
     fmt
 
+let show_err fmt = show_message `Error fmt
+
+let show_warn fmt = show_message `Warn fmt
+
+let show_info fmt = show_message `Info fmt
```

Comment:
These seem unnecessary. `show_message` is already a shortcut to the `Window.show...Message` functions.